roles/edpm_ceph_client_files/tasks/main.yml

```diff
@@ -14,17 +14,6 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-- name: Fail if edpm_ceph_client_files_source is missing
```

Comment:
In general this looks good to me, we don't have to fail and propagate the failure state to the `dataplane-operator` (the AEE execution Status).